addons/test_mail/static/tests/tracking_value_tests.js

```diff
@@ -52,41 +52,41 @@ QUnit.test('basic rendering of tracking value (float type)', async function (ass
     await click('.o_form_button_save');
     assert.containsOnce(
         document.body,
-        '.o_TrackingValue',
+        '.o_MessageTrackingValue',
         "should display a tracking value"
     );
     assert.containsOnce(
         document.body,
-        '.o_TrackingValue_fieldName',
+        '.o_MessageTrackingValue_changedField',
         "should display the name of the tracked field"
     );
     assert.strictEqual(
-        document.querySelector('.o_TrackingValue_fieldName').textContent,
+        document.querySelector('.o_MessageTrackingValue_changedField').textContent,
         "Float:",
         "should display the correct tracked field name (Float)",
     );
     assert.containsOnce(
         document.body,
-        '.o_TrackingValue_oldValue',
+        '.o_MessageTrackingValue_oldValue',
         "should display the old value"
     );
     assert.strictEqual(
-        document.querySelector('.o_TrackingValue_oldValue').textContent,
+        document.querySelector('.o_MessageTrackingValue_oldValue').textContent,
         "12.30",
         "should display the correct old value (12.30)",
     );
     assert.containsOnce(
         document.body,
-        '.o_TrackingValue_separator',
+        '.o_MessageTrackingValue_arrow',
```

Comment:
What is the reason to rename separator in arrow specifically?

Reply:
that was because of the rebase problem.. I will change that.